### PR TITLE
[Rule Tuning] AWS RDS Snapshot Export and AWS RDS Instance Created

### LINKED ISSUE #1513: [Rule Tuning] AWS RDS Snapshot Export and AWS RDS Instance Created
## Link to rule
https://github.com/elastic/detection-rules/blob/main/rules/integrations/aws/exfiltration_rds_snapshot_export.toml

## Description

It looks like I never added my name to author when created this originally.

Original Pull Request: 
#1269, #1270

### PR DESCRIPTION
**Pull request by austinsonger**
## Issues

Resolves #1513

Original

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/integrations/aws/exfiltration_rds_snapshot_export.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ updated_date = "2021/07/20"
 integration = "aws"
 
 [rule]
-author = ["Elastic"]
+author = ["Elastic","Austin Songer"]
```
Comment (w0rk3r) [inline rules/integrations/aws/exfiltration_rds_snapshot_export.toml:8]:
```suggestion
author = ["Elastic", "Austin Songer"]
```

File: rules/integrations/aws/persistence_rds_instance_creation.toml [hunk 2]
```diff
@@ -5,7 +5,7 @@ updated_date = "2021/07/20"
 integration = "aws"
 
 [rule]
-author = ["Elastic"]
+author = ["Elastic","Austin Songer"]
```
Comment (w0rk3r) [inline rules/integrations/aws/persistence_rds_instance_creation.toml:8]:
```suggestion
author = ["Elastic", "Austin Songer"]
```

**Review by w0rk3r — APPROVED** [review]
Just a small change, then LGTM

**Review by brokensound77 — APPROVED** [review]
Bump `updated_date` and LGTM 👍 

Thanks @austinsonger, attribution is important

**Comment by austinsonger** [comment]
@brokensound77 

> Bump `updated_date` and LGTM 👍
> 
> Thanks @austinsonger, attribution is important

Done.

**Pull request merged**